### PR TITLE
Jhonson Algorithm in PHP

### LINKED ISSUE #2206: Johnson Algorithm
- [x] C 
- [ ] C++ - @Aakansha99
- [x] C# - @ritish099
- [ ] CoffeeScript
- [x] Dart - @Swarnimashukla
- [ ] Go - @BhardwajShweta25
- [ ] Java - @Neelaksh-Singh
- [ ] JS - @thakran14
- [x] Kotlin - @ritish099
- [x] Php - @ritish099
- [ ] Python - @BishalKrChanda
- [ ] Readme - @AdityaSingh17
- [ ] Ruby
- [ ] Rust
- [ ] TypeScript

### PR DESCRIPTION
**Pull request by ritish099**
Fixes #2206

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Implemented Jhonson Algorithm in PHP

#### Languages Used:
- PHP

#### Files Added:
- Jhonson_Algorith.php

Thanks!

### CODE REVIEW

**Comment by ritish099** [comment]
Mam @asha15 , Pls check it out

File: Jhonson_Algoritm/Jhonson_Algorithm.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<?php
```
Comment (asha15) [inline Jhonson_Algoritm/Jhonson_Algorithm.php:1]: please a line space after this

File: Jhonson_Algoritm/Jhonson_Algorithm.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        $j=intval(fgets(STDIN));
+        $c=intval(fgets(STDIN));
+        $adj[$i][$j] = $cost[$i][$j] = $c;
+    }
+    
+    for ($i = 1; $i = $vert; $i++)
+        for ($j = 1; $j = $vert; $j++)
+        {
+            if ($adj[$i][$j] == 0 && $i != $j)
+                //If its not a edge put infinity
```
Comment (asha15) [inline Jhonson_Algoritm/Jhonson_Algorithm.php:23]: remove the spaces you kept at the beginning

File: Jhonson_Algoritm/Jhonson_Algorithm.php [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    // Output the resultant matrix
+    for ($i = 1; $i = $vert; $i++)
+    {
+        for ($j = 1; $j = $vert; $j++)
+        {
+            if ($adj[$i][$j] != $INF)
+            echo implode(', ',$adj[$i][$j]);
+        }
+        echo "".PHP_EOL;
+    }
```
Comment (asha15) [inline Jhonson_Algoritm/Jhonson_Algorithm.php:42]: please remove the spaces at the beginning

Comment (asha15) [inline Jhonson_Algoritm/Jhonson_Algorithm.php:42]: keep a line space after this

File: Jhonson_Algoritm/Jhonson_Algorithm.php [hunk 4]
```diff
@@ -0,0 +1,57 @@
+<?php
+    $INF = 999999;
+    $adj = array();
+    $cost = array();
```
Comment (asha15) [inline Jhonson_Algoritm/Jhonson_Algorithm.php:4]: keep a line space after this

File: Jhonson_Algoritm/Jhonson_Algorithm.php [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    echo "Enter no of vertices".PHP_EOL ;
+    $vert=intval(fgets(STDIN));
+    echo "Enter no of Edges".PHP_EOL;
+    $edge=intval(fgets(STDIN));
```
Comment (asha15) [inline Jhonson_Algoritm/Jhonson_Algorithm.php:8]: keep space around equal

File: Jhonson_Algoritm/Jhonson_Algorithm.php [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+    echo "Enter the EDGE cost".PHP_EOL ;
```
Comment (asha15) [inline Jhonson_Algoritm/Jhonson_Algorithm.php:9]: keep a line space after this

File: Jhonson_Algoritm/Jhonson_Algorithm.php [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+    for ($k = 1; $k = $edge; $k++)
+    {
+        // the input and store it into adj and cost matrix
+        $i=intval(fgets(STDIN));
+        $j=intval(fgets(STDIN));
+        $c=intval(fgets(STDIN));
```
Comment (asha15) [inline Jhonson_Algoritm/Jhonson_Algorithm.php:15]: keep space around equal

File: Jhonson_Algoritm/Jhonson_Algorithm.php [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+        $adj[$i][$j] = $cost[$i][$j] = $c;
+    }
+    
+    for ($i = 1; $i = $vert; $i++)
+        for ($j = 1; $j = $vert; $j++)
+        {
+            if ($adj[$i][$j] == 0 && $i != $j)
+                //If its not a edge put infinity
+                $adj[$i][$j] = $INF; 
+        }
```
Comment (asha15) [inline Jhonson_Algoritm/Jhonson_Algorithm.php:25]: keep a line space after this

File: Jhonson_Algoritm/Jhonson_Algorithm.php [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+    for ($k = 1; $k = $vert; $k++)
+        for ($i = 1; $i = vert; $i++)
+            for ($j = 1; $j = $vert; $j++)
+                //Finding the minimum
+                //find minimum path from i to j through k
+                $adj[$i][$j] = min(($adj[$i][$k] + $adj[$k][$j]),$adj[$i][$j] );
+    echo "The distance matrix of the graph";
```
Comment (asha15) [inline Jhonson_Algoritm/Jhonson_Algorithm.php:32]: keep a line space after this

**Comment by asha15** [comment]
Good job!

please squash the commits

**Comment by ritish099** [comment]
Mam @asha15 , Check it out

**Pull request merged**